test/TankTests.cpp

```diff
@@ -37,10 +39,8 @@ struct TankTestData {
   unsigned int health = 100;
 
   Tank create_tank(std::unique_ptr<testing::NiceMock<EngineMock>>&& engine) {
-    return {0,
-            0,
-            mTextures,
-            std::move(engine),
+    return {0, 0, mTextures, std::move(engine),
+            emptySound,
```

Comment:
This is the place when Sound will be copied. 

If you want to use "fakeSound", I believe you will need to add some flag to the "Sound" class (and then - empty constructor will be OK). Or maybe you don't need a flag, but simply default buffer will not play anything (and comment from my last Pull Request about removing default constructor was incorrect? :( )

Reply:
I restored default constructor in order to make possible compilation of test.

Reply:
If i added throwing exception if file does not exist, I want to add test if it works.